src/playlist-controller.js

```diff
@@ -232,6 +235,14 @@ export class PlaylistController extends videojs.EventTarget {
     // we don't have to handle sourceclose since dispose will handle termination of
     // everything, and the MediaSource should not be detached without a proper disposal
 
+    if (this.hasManagedMediaSource_) {
+      this.handleStartStreaming_ = this.handleStartStreaming_.bind(this);
```

Comment:
You can add the content of this if statement to the above pre-existing if statement. All nits, but saves a few lines

Reply:
You mean to the `if (experimentalUseMMS && window.ManagedMediaSource) {...}` block? It is more concise but  I opted against it because to my eye it made the code slightly less organized by putting the new `addEventListener` calls inside the block where the MediaSource is set rather than in the subsequent step/section where the other MediaSource event handlers are added. Minor issue, I'm happy to change it

Reply:
Yeah, all my comments were basically to save some lines of code by moving this code to where you set the `hasManagedMediaSource` flag and making the flag unnecessary, but fine with me if it keeps it cleaner in your eyes. 👍